lib/framework/data_context.dart

```diff
@@ -543,6 +546,96 @@ class EnsembleStorage with Invokable {
   }
 }
 
+/// singleton handling utils
+class EnsembleUtils with Invokable {
+  static final EnsembleUtils _obj = EnsembleUtils._();
+  EnsembleUtils._();
+  factory EnsembleUtils() => _obj;
+
+  @override
+  Map<String, Function> getters() => {'getCountries': () => allCountries};
+
+  @override
+  Map<String, Function> methods() => {
+        'getCountries': (value) {
```

Comment:
Please review the jiras requirements:
#867 - support `getCountries()` and `getCountry(<alpha-2>)`
#871 - support `findCountry(<search string>)`